scripts/distributables/macos-concordium-client.Jenkinsfile

```diff
@@ -29,26 +30,16 @@ pipeline {
             agent { label 'mac' }
             steps {
                 sh '''\
-                    # Ensure using custom version of ghc
-                    # Stack is refusing for some reason to use the ghc installed by ghcup :/ so I'm installing it via stack.
-                    rm -rf ~/.stack/*
-                    cat <<'EOF' > ~/.stack/config.yaml
-                    setup-info:
-                      ghc:
-                         macosx-custom-integer-simple:
-                             8.8.4:
-                                  url: "https://s3-eu-west-1.amazonaws.com/static-libraries.concordium.com/ghc-8.8.4-x86_64-apple-darwin.tar.xz"
+                    # Install correct version of rust.
+                    rustup default $RUST_VERSION
 
-                    ghc-variant: integer-simple
-                    EOF
-                    
+                    # due to a bug in stack and flag propagation we have to do this hack
                     sed -i '' "s/default: False/default: True/g" deps/concordium-base/package.yaml
+                    sed -i '' "s/default: False/default: True/g" deps/concordium-base/concordium-base.cabal
+
+                    # At present concordium-client does not build in a static build with any version of ghc more recent than 8.8.4.
+                    stack build --compiler=ghc-8.8.4 --flag concordium-client:-middleware
```

Comment:
Maybe use `GHC_VERSION` instead of hardcoding?

Reply:
The thing is that it is deliberately hardcoded since it does not work with other versions.